### PR TITLE
cleans up datawarehouse deployment

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- cleans up datawarehouse deployment

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

**Review by Baalmart — APPROVED** [review]
LGTM.
Thanks @123MwanjeMike

File: k8s/data-warehouse/stage-data-warehouse-api.yaml [hunk 1]
```diff
@@ -82,70 +82,6 @@ spec:
...
-metadata:
-  name: airqo-stage-datawarehouse-msg-broker-hpa
-spec:
-  scaleTargetRef:
-    apiVersion: apps/v1
-    kind: Deployment
-    name: airqo-stage-datawarehouse-msg-broker
-  minReplicas: 1
-  maxReplicas: 2
-  metrics:
```
Comment (NoahNsimbe) [inline k8s/data-warehouse/stage-data-warehouse-api.yaml:148]: Hi @123MwanjeMike looks like you are removing the deployment manifest for the message broker but you did not delete the entire metircs section..

Comment (123MwanjeMike) [reply k8s/data-warehouse/stage-data-warehouse-api.yaml:148]: Hi @NoahNsimbe 
Those are metrics for data warehouse deployment hpa.
https://github.com/airqo-platform/AirQo-api/blob/b4b99a61557b5bae7ca46708711cf3cdbd3ac040/k8s/data-warehouse/stage-data-warehouse-api.yaml

Comment (NoahNsimbe) [reply k8s/data-warehouse/stage-data-warehouse-api.yaml:148]: Great. Thanks

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**